Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = () => {
+    let today = new Date().toISOString().slice(0, 10);
```

Comment:
This is not the exact format that the design file wanted. You can just out moment.js to help with that! Another way to do it is to use unix timestamp and use some built-in methods with Date() to grab the month and day. If you want help writing it from scratch let me know.